### PR TITLE
Updated code samples to reflect App.js instead of index.js

### PR DESCRIPTION
**Pull request by jacebenson**
Hey, I went ahead and updated the comments on the snippets in the GoTure_Auth.md and authentication.md.  I also updated it in the customIndex.md, however that wasn't called out in the issue.

### CODE REVIEW

**Comment by jacebenson** [comment]
This PR is for #634

File: cookbook/GoTrue_Auth.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ yarn redwood generate auth goTrue
 By specifying `goTrue` as the provider, Redwood automatically added the necessary GoTrue-JS config to our index.js. Let's open up `web/src/index.js` and inspect. You should see:
```
Comment (thedavidprice) [inline cookbook/GoTrue_Auth.md:44]:
```suggestion
By specifying `goTrue` as the provider, Redwood automatically added the necessary GoTrue-JS config to our App.js. Let's open up `web/src/App.js` and inspect. You should see:
```

File: docs/customIndex.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ yarn rw setup custom-web-index
 This generates a file named `index.js` in `web/src` that looks like this:
 
 ```js
-// web/src/index.js
+// web/src/App.js
```
Comment (thedavidprice) [inline docs/customIndex.md:14]:
```suggestion
// web/src/index.js
```

**Review by thedavidprice — CHANGES_REQUESTED** [review]
@jacebenson There were a couple of index.js --> App. js changes inline that were missed and I fixed.

The next change is to update the code in the App.js file itself. This is no longer relevant:
```js
ReactDOM.render(
  <FatalErrorBoundary page={FatalErrorPage}>
    <AuthProvider client={auth0} type="auth0">
      <RedwoodApolloProvider>
        <Routes />
      </RedwoodApolloProvider>
    </AuthProvider>
  </FatalErrorBoundary>,
  document.getElementById('redwood-app')
)
```

File: cookbook/GoTrue_Auth.md [hunk 3]
```diff
@@ -38,21 +38,20 @@ Now you should see an Identity API endpoint, e.g. `https://my-bodacious-app.netl
 Let's start by installing the required packages and generating boilerplate code and files for Redwood Auth, all with this simple [CLI command](/docs/cli-commands#generate-auth):
 
 ```bash
-yarn redwood generate auth goTrue
```
Comment (jacebenson) [inline cookbook/GoTrue_Auth.md:41]: When generating the code for the blocks before I copied/pasted this in and it didnt work.. So updated this to work.

File: cookbook/GoTrue_Auth.md [hunk 4]
```diff
@@ -38,21 +38,20 @@ Now you should see an Identity API endpoint, e.g. `https://my-bodacious-app.netl
...
+yarn redwood setup auth goTrue
```
Comment (jacebenson) [inline cookbook/GoTrue_Auth.md:41]: Corrected that here

**Comment by jacebenson** [comment]
I did entirely miss the code blocks on the docs/authentication.md. I've since updated that file and the gotrue cookbook file to match the generators.

I went ahead and tested each command on a local nearly empty repo and copied the generated code where it applied.

File: docs/authentication.md [hunk 5]
```diff
@@ -153,28 +160,40 @@ To get your application keys, only complete the ["Configure Auth0"](https://auth
...
+import FatalErrorPage from 'src/pages/FatalErrorPage'
+import Routes from 'src/Routes'
+
+import './index.css'
 
 const auth0 = new Auth0Client({
   domain: process.env.AUTH0_DOMAIN,
   client_id: process.env.AUTH0_CLIENT_ID,
-  redirect_uri: 'http://localhost:8910/',
+  redirect_uri: process.env.AUTH0_REDIRECT_URI,
```
Comment (thedavidprice) [inline docs/authentication.md:177]: great addition and comments!

**Review by thedavidprice — APPROVED** [review]

**Comment by thedavidprice** [comment]
@jacebenson woah, I can tell you really spent some time digging into this one and getting the details right! Thank you so much. This was a greatly needed improvement.

**Pull request merged**